### DISCUSSION TITLE
technical onboarding

### DISCUSSION THREAD

**Discussion by adityaSkhorne2401** [microsoft/SATechnicalOnboarding#74] [Questions]
hello everyone, my name is Aditya and I am a new student ambassador.I am unable to complete my technical onboarding because I haven't  found the required github repository.so please help me to complete

**Comment by AnthonyByansi** [comment]
Hello @adityaSkhorne2401 the same repo you've opened up a discussion is the same where you get started! 
You can kick off with the [Instructions here ](https://github.com/microsoft/SATechnicalOnboarding/blob/main/technical-onboarding-instructions.md)

**Reply by adityaSkhorne2401** [reply]
I need help for technical onboarding

**Reply by Abhayg444** [reply]
> I need help for technical onboarding

### Portfolio 

This is the template repository which you can use to make your portfolio site : [Protfolio](https://github.com/education/codespaces-project-template-js). You just need to clone it and make the necessary changes according to your desire. You can also use other templates to create your own portfolio. The steps are already mentioned.

### Pull request task 

You have to use this repository it has al the steps needed to complete the task [Pull request ](https://github.com/skills/review-pull-requests). All the necessary steps are already given make sure to follow them to avoid mistakes.

**Comment by JanninaRM** [comment]
Hi my name is Jannina, would you be grateful if you could confirm if the github account should be created with ambassador mail or can it also be an account that I have with another mail? thanks a lot.

**Reply by Abhayg444** [reply]
You can use your personal github account and link the mlsa email id in the secondary email section. Additional for the verification of the technical onboarding process you have to give your github repository link for the pull request task.

**Reply by AnthonyByansi** [reply]
Thanks for your help @Abhayg444

**Reply by JanninaRM** [reply]
Thank you @Abhayg444, happy new year!

**Reply by Abhayg444** [reply]
Happy New Year 🌟 to you too..!!

**Comment by Maryam-html-css-javascript** [comment]
hello everyone can anyone tell after completion of onboarding  is there any mail which tells that our onboarding accepts or not
